wordpress-mobile/WordPress-Shared-iOS
# Replace AFNetworking dependency for NSURLSession

This PR replaces the use of AFNetworking for NSURLSession on the WPImageSource class.

It also updates the current tests to use XCTestExpectations and to use the latests version OHHTTPStubs in order to support NSURLSession methos.

How to test:
 * Run the unit tests and make sure they are ok.
 * Integrate on the main app and check stats, gravatar, the WordPress media library, and the reader.
 
Needs Review: @astralbodies
